doc/ble.md

```diff
@@ -70,34 +74,37 @@ When the service does not exist in the BLE specification, InfiniTime implements
 
 The following custom services are implemented in InfiniTime:
 
- - Since InfiniTime 0.8:
-    * Music Service : 00000000-78fc-48fe-8e23-433b3a1942d0
- 
+- Since InfiniTime 0.8:
 
- - Since InfiniTime 0.11:
-     * [Navigation Service](NavigationService.md) : 00010000-78fc-48fe-8e23-433b3a1942d0
+  - Music Service : 00000000-78fc-48fe-8e23-433b3a1942d0
 
+- Since InfiniTime 0.11:
+
+  - [Navigation Service](NavigationService.md) : 00010000-78fc-48fe-8e23-433b3a1942d0
 
 - Since InfiniTime 0.13
-    * Call characteristic (extension to the Alert Notification Service): 00020001-78fc-48fe-8e23-433b3a1942d0
 
+  - Call characteristic (extension to the Alert Notification Service): 00020001-78fc-48fe-8e23-433b3a1942d0
 
 - Since InfiniTime 1.7:
-    * [Motion Service](MotionService.md): 00030000-78fc-48fe-8e23-433b3a1942d0
 
+  - [Motion Service](MotionService.md): 00030000-78fc-48fe-8e23-433b3a1942d0
 
 - Since InfiniTime 1.8:
-    * [Weather Service](/src/components/ble/weather/WeatherService.h): 00040000-78fc-48fe-8e23-433b3a1942d0
+  - [Weather Service](/src/components/ble/weather/WeatherService.h): 00040000-78fc-48fe-8e23-433b3a1942d0
```

Comment:
are the added newlines necessary? I think keeping the enumerations together would be rendered just fine (less space between bullet points in the final render). For example:

```md
- Since IT 0.1.2
  - some service: `000-111`
- Since IT 0.1.2
  - some service: `000-111`
```
produces:

- Since IT 0.1.2
  - some service: `000-111`
- Since IT 0.1.2
  - some service: `000-111`

Whereas an example with spaces between them

```md
- Since IT 0.1.2

  - some service: `000-111`

- Since IT 0.1.2

  - some service: `000-111`
```
produces:

- Since IT 0.1.2

  - some service: `000-111`

- Since IT 0.1.2

  - some service: `000-111`

---

and while we're at it I recommend adding backticks to the BLE Service IDs as done in my examples above

Reply:
Adding the newlines didn't change the rendering, because there were empty lines in between. I'd rather leave it rendering the same way for now.

```
- Since IT 0.1.2
  - some service: `000-111`

- Since IT 0.1.2
  - some service: `000-111`
```